README.md

```diff
@@ -4,13 +4,60 @@ HasCore: HasGeek API
 ![Build status](https://secure.travis-ci.org/hasgeek/hascore.png)
 [![Coverage Status](https://coveralls.io/repos/hasgeek/hascore/badge.png?branch=master)](https://coveralls.io/r/hasgeek/hascore?branch=master)
 
-Hascore provides helper API methods including the networkbar that is shown
-on all sites, and API endpoints for geolocation and language analysis.
+Hascore provides helper API methods including the networkbar that is shown on all sites, and API endpoints for geolocation and language analysis.
 
-Hascore requires NLTK. You will need to install NLTK's corpus data.
-Be warned, this is about 1.8 GB:
+-----
+### NLTK
+
+Hascore requires NLTK. You will need to install NLTK's corpus data. Be warned, this is about 1.8 GB:
 
     $ python -m nltk.downloader all
+    
+### Postgres
+
+Hascore uses Postgres >=9.4 and Redis server for development. To set up a Postgres DB:
+
+On OS X using the [Postgres App](http://postgresapp.com):
+
+    $ # Add Postgres app to the path if it's not already in there
+    $ export PATH="/Applications/Postgres.app/Contents/Versions/9.4/bin:$PATH"
+    $ # Make the user and database
+    $ createuser -d hascore 
+    $ createdb -O hascore -W hascore
+
+On any Linux distribution:
+
+    $ sudo -u postgres createuser -d hascore
+    $ sudo -u postgres createdb -O hascore hascore
+    
+* Edit the `\instance\setting-sample.py` to change the following variables: `SQLALCHEMY_DATABASE_URI` to `postgres://hascore:YOUR_PASSWORD_HERE@localhost/hascore`, `LASTUSER_CLIENT_ID` = `id` and `LASTUSER_CLIENT_SECRET` = `pwd`, where `id` and `pwd` are the authorization tokens received after registering with [HasGeek Auth](https://auth.hasgeek.com/).
```

Comment:
Formatting nitpicks: 

1. No need to mark this is as list item (star * syntax), its a paragraph.
2. Mark entire text that goes into the file with backtick and not variable and value sepately. For example, I would revise the above to: 

`LASTUSER_CLIENT_ID = 'CLIENT_ID'` where CLIENT_ID is to be replaced with your actual Client ID registered from auth.hasgeek.com.

Try not to change the example value to something else, it will confuse the user. If Client ID is what you want them to replace in between quotes, then indicate so in capitals. If Client secret is what you want them to fill in, then write CLIENT_SECRET.